src/main/resources/application.properties

```diff
@@ -1,14 +1,20 @@
 spring.application.name=book-store
 server.servlet.context-path=/api
 
-spring.datasource.url=jdbc:mysql://localhost:3306/book_store?serverTimeZone=UTC
+spring.datasource.url=jdbc:mysql://localhost:3306/book_store\
+  ?serverTimeZone=UTC&createDatabaseIfNotExist=true
 spring.datasource.username=
 spring.datasource.password=
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 
 spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
 spring.jpa.open-in-view=false
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQLDialect
+
+spring.liquibase.enabled=true
+springdoc.api-docs.enabled=true
+springdoc.swagger-ui.enabled=true
```

Comment:
It's default values

Reply:
I want to add these properties so I don't forget about their existence